01-messaging.md

```diff
@@ -23,6 +23,13 @@ A node MUST NOT send an evenly-typed message not listed here without prior negot
 A node MUST ignore a received message of unknown type, if that type is odd.
 A node MUST fail the channels if it receives a message of unknown type, if that type is even.
 
+The messages are grouped logically into 4 groups by their most significant set bit:
+
+ - Setup & signalling (types `0`-`31`): comprises setup of the cryptographic transport, communication of supported features and error reporting. These are described below.
```

Comment:
You can remove "setup of the cryptographic transport," as that doesn't use these codes.